lib/event_sourcery/event_processing/event_stream_processor.rb

```diff
@@ -40,7 +40,7 @@ def process(event)
           end
           @_event = nil
         rescue
-          raise EventProcessingError.new(event)
+          raise EventProcessingError.new(event), "Processor: #{processor_name.to_s}\nEvent: #{event.inspect}"
```

Comment:
I'm not sure about `event.inspect` in production... events might contain things shouldn't be logged or sent to error reporters. Looks very useful for dev/test though.

Reply:
Good call. I've pushed a fixup commit (d0a46be) that does a custom inspect and I've moved the logic into the `EventProcessingError` class.